<?xml version="1.0"?>
<launch>
    <arg name="can_channel" default="can0" />
    <arg name="can_type" default="socketcan" />

    <param name="can_channel" value="$(arg can_channel)" />
    <param name="can_type" value="$(arg can_type)" />
    <param name="dbc_path" value="$(find pix_driver)/src/self_driving.dbc" />

    <node name="pixkit_status" 
            pkg="pix_driver" type="pixkit_status.py" 
            required="true" />

    <node name="pixkit_control"
            pkg="pix_driver" type="pixkit_control.py" 
            required="true"
            output="screen" />

    <node name="demo"
            pkg="pix_driver" type="demo.py" 
            required="true" 
            output="screen" />
</launch>
